web/tools.html

```diff
@@ -181,7 +192,7 @@ <h4>Tools</h4>
         <script>
             function openCloseMenu() {
                 var toolBar = document.getElementById("toolBar");
-                if (toolBar.style.width === "0px") {
+                if (toolBar.style.width === "0px" || toolBar.style.width === "") {
```

Comment:
I have seen the page content that is to be shifted aside when the menu bar is opened wrapped into a class of its own before in order to avoid the overlap, so the main content width can be changed with an alteration to the style similar to this. Is there a different way to achieve the same effect that we could implement?